### DISCUSSION TITLE
[Where] does brew store meta information?

### DISCUSSION THREAD

**Discussion by jsejcksn** [Homebrew/discussions#50] [Everyday usage]
...for example, the timestamp of the last time that `brew update` was executed (or formulae were updated)

**Comment by jonchang** [comment]
This can be discovered by reading the source code. In this case, `brew` checks the mtime of the core tap directory.

**Reply by jsejcksn** [reply]
Thanks for responding @jonchang, although I'm not positive about what you're referencing. Is this what you mean?

https://github.com/Homebrew/brew/blob/e30ed42439f4cca705be54aa14193f6f0a05420d/Library/Homebrew/brew.sh#L540-L552
